src/core/utils/multicall.ts

```diff
@@ -81,7 +85,7 @@ export class MulticallQueue {
     const batchSize = callsToProcess.length
     logger.debug({ batchSize }, 'Sending multicall batch ')
 
-    const results = await this.multicallContract.callStatic.aggregate3(
```

Comment:
They changed the way this is called.

Reply:
Ok good, in solidity I think they call it `staticcall` I'm glad ethers changed